src/services/categoryService.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const {
+  createCategory,
+  getCategoryById,
+  updateCategory,
+  deleteCategory,
+} = require('../dbQueries');
+
+const createCategoryService = async(name) => {
+  return createCategory(name);
+};
+
+const getCategoryByIdService = async(id) => {
+  return getCategoryById(id);
+};
+
+const updateCategoryService = async(id, name) => {
+  return updateCategory(id, name);
+};
+
+const deleteCategoryService = async(id) => {
+  return deleteCategory(id);
+};
```

Comment:
prettier?

Reply:
it may look identical because adding spaces between async and (param) is all the formatter did here